hack/jenkins/minikube_cross_build_and_upload.sh

```diff
@@ -26,6 +26,13 @@ gsutil cp gs://minikube-builds/logs/index.html gs://minikube-builds/logs/${ghprb
 # Build all platforms (Windows, Linux, OSX)
 make cross
 
+## If there are ISO changes, build the ISO and upload it to be used in the integration tests
+if git diff --name-only origin/master...HEAD | grep deploy/iso/minikube-iso > /dev/null; then
```

Comment:
It might be nice to save the output of this and log it if we think something changed.